appointments/appointment-routes.js

```diff
@@ -0,0 +1,80 @@
+const uuid = require("uuid");
+const AWS = require("aws-sdk");
+require("dotenv").config();
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION,
+});
+const dynamodb = new AWS.DynamoDB.DocumentClient();
+
+module.exports = (app) => {
+  app.get("/getAppointments/:doctorEmail", (req, res) => {
+    let { doctorEmail } = req.params;
```

Comment:
the endpoint is not protected against JWT. before storing the email in dynamo db do you verify if the doctor is present on the blockchain?

Reply:
What is the middleware function used to validate the JWT?
Also, the doctor would have to login to the application to enter his availability for the week. 
We can also have the admin of the HC to enter the availability fo the doctor.
Either ways, the availability table in dynamodb will have only the registered doctors